improver/utilities/cube_manipulation.py

```diff
@@ -791,7 +791,7 @@ def height_of_maximum(
             A cube of the maximum value over the height coordinate.
         find_lowest:
             If true then the lowest maximum height will be found (for cases where
-            there are two heights with the maximum vertical velocity.) Otherwise the highest
+            there are two heights with the maximum height velocity.) Otherwise the highest
```

Comment:
We've nearly left this file unchanged. Last thing is that whilst we are talking about height levels, the velocity discussed is the velocity in the z-dimension, i.e. the vertical velocity. If you can put this back as shown here then this file will be exactly as it was before.
```suggestion
            there are two heights with the maximum vertical velocity.) Otherwise the highest
```

If in future you want to completely undo the changes to a file, the easiest way is to go and get the file back from a commit before you made any changes. You can do this as follows.

1. Use `git log` to have a look at the commit history of your branch.
2. Copy the hash (the long number/letter string) that identifies the commit you want.
3. Checkout the file that you want to reset from that commit. In this case, assuming your in the head directory (`~/improver`) it would be this:

```
git checkout 347a8a437c1284e04d01253828cac48ce11487e4 -- improver/utilities/cube_manipulation.py
```

This works fine if you've got a nice linear commit history, or even if you've merged in master, as long as no one else is editing the file you're interested in.

Reply:
Thanks Ben, that is useful to know for future! I have made the final change.